lib/kennel/models/monitor.rb

```diff
@@ -38,20 +38,20 @@ class Monitor < Record
         message: -> { "\n\n#{project.mention}" },
         escalation_message: -> { DEFAULT_ESCALATION_MESSAGE.first },
         renotify_interval: -> { project.team.renotify_interval },
-        warning: -> { nil },
```

Comment:
rubocop didnt like `{ nil }`

Reply:
seems weird since the rubocop version is hardcoded ... so maybe you ran it with a different version of rubocop or no .rubocop.yml ?

Reply:
.... also this exists in multiple files, so please restore. If we decide to change that it should be it's own PR.

Reply:
ya i did run directly `rubocop` without `bundle exec` at the front. i can put back the nil